### PR TITLE
r.suitability.regions, new addon

### PR DESCRIPTION
**Pull request by ecodiv**
Addon to identify contiguous regions of high suitability, based on an input suitability map. Based r.reclass.area, r.clump and r.area.

### CODE REVIEW

**Review by veroandreo — COMMENTED** [review]
Cool addon @ecodiv !! 
I only reviewed the manual page

File: grass7/raster/r.suitability.regions/r.suitability.regions.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+<p><img src="option03.png"><i>Figure 3: Like figure 2, but considering 
+raster cells with suitability 0 (mostly roads) as absolute barriers. 
+Diagonally connected raster cells are not considered to form a 
+contiguous region.</i></p>
+
+<p>Note that for line elements like roads, results may differ if the 
+option to include the diagonal neighbors when defining clumps is 
+selected. For example, in figure 4, diagonally connected cells are 
+considered as neighbors. So the areas above and below the road are 
+connected at among others the marked locations.
```
[the remark this answers is not in the document]
Comment (ecodiv) [reply grass7/raster/r.suitability.regions/r.suitability.regions.html:102]: I replaced the last sentence with: "As a consequence, the suitable areas on both sides of the road are considered to be part of the same region. I.e., the road does not act as a barrier here."

File: grass7/raster/r.suitability.regions/r.suitability.regions.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+gaps (areas within a suitable region) of 500 hectares or less were 
+included in the final selection (middle). The right map shows the 
+suitable areas within the selected regions (green) and the filled gaps 
+(yellow).</i></p>
+
+<p>Selecting this option will generate a second map which shows the 
+these 'filled patches'. This makes it easier to  e.g., inspect the 
+feasibility or desirability to actually include these areas in a 
+protected areas. It furthermore makes it easier to calculate the area 
+of the total area of the region and the area within the region with
```
[the remark this answers is not in the document]
Comment (ecodiv) [reply grass7/raster/r.suitability.regions/r.suitability.regions.html]: Removed the sentence, was more confusing than useful

**Comment by ecodiv** [comment]
Thanks @veroandreo , good improvements, accepted all proposed changes

**Comment by ecodiv** [comment]
@veroandreo , you probably know this. What is the proceedure, add-on code needs to be reviewed and then merged by somebody of the development team or other add-on contributers?

**Comment by veroandreo** [comment]
You have permission to merge in addons, no? So, just go ahead and merge, or if you prefer that someone else reviews it, maybe ask for their review explicitly.

**Comment by ecodiv** [comment]
Hi @wenzeslaus, I'm not very well versed in github. Would it be the easiest if I close this pull request and create a new one for the grass7 branch?

**Comment by wenzeslaus** [comment]
> Would it be the easiest if I close this pull request and create a new one for the grass7 branch?

Since this a new module (new files), it might be easiest for you if you redo this in a new PR.

You should also use a new branch, e.g., add-suitability-regions, to do that. Generally, you used the master branch to commit the changes. Since we are just trashing it, it would not be a problem here. In future, you don't want to use the grass7 branch to commit your changes (it makes it more difficult to manage your local repo).

If you feel Git-adventurous, you could just change the base branch here (_Edit_ button), merge with upstream/grass7 (as you previously did with upstream/master), and then just move your files to the right place (git mv). It should be relatively straightforward. Generally, this should be less work then doing another PR. If you make a backup of your files (i.e., all your changes), the worst thing which can happen is that you will have to delete your local repo (clone) and your fork and start the contributing process over (see the CONTRIBUTING.md file, just replace master with grass7).

**Comment by ecodiv** [comment]
Closing this, will redo a pull request on the grass7 branch

**Pull request closed without merging**